crates/cxx-qt-lib/src/qml/qqmlapplicationengine.rs

```diff
@@ -79,13 +82,33 @@ mod ffi {
         ) -> Pin<&mut QQmlEngine>;
     }
 
+    #[cfg(any(cxxqt_qt_version_at_least_7, cxxqt_qt_version_at_least_6_5))]
+    unsafe extern "C++" {
```

Comment:
This #[cfg] doesn't work I guess, so Qt5 and <6.5 builds will fail. An easy solution would be to separate this into a different cxx::bridge as suggested in [this issue](https://github.com/dtolnay/cxx/issues/989) but that seems extreme?

Reply:
Fully support for cfg flags everywhere comes in #1133 and #1132 which i plan to look at next.

For now you could try instead of putting the cfg flag on the extern block itself put it on the method as this is passed through to CXX.

Reply:
That does work, but the QAnyStringView inclusion is hitting the same problems as seen in #1141 :(